p2p/src/authenticated/mod.rs

```diff
@@ -471,13 +471,43 @@ mod tests {
 
     #[test_traced]
     fn test_tokio_connectivity() {
```

Comment:
The purpose of this test is to ensure `p2p` works with the `tokio` runtime. I think this is a good test but it should live in `runtime` (it has nothing to do with `p2p`).